pkg/minishift/registration/redhat_test.go

```diff
@@ -27,14 +27,10 @@ import (
 
 var (
 	param = &RegistrationParameters{
-		Username:        "foo",
-		Password:        "foo",
-		ProxyUsername:   "foo",
-		ProxyPassword:   "foo",
-		ProxyServer:     "foo",
-		ProxyServerPort: "foo",
+		Username: "foo",
+		Password: "foo",
 	}
-	expectedCMDRegistration = fmt.Sprintf("sudo subscription-manager register --auto-attach --username %s --password %s ",
+	expectedCMDRegistration = fmt.Sprintf("sudo -E subscription-manager register --auto-attach --username %s --password %s ",
```

Comment:
"-E" is used to preserve the environment variable for the shell we need it here so our proxy env variable can be picked easily.